lib/services/oauth/oauth_server.go

```diff
@@ -0,0 +1,68 @@
+package oauth
+
+import (
+	"context"
+	"os"
+
+	"github.com/markgravity/golang-ic/helpers/log"
+
+	"github.com/go-oauth2/oauth2/v4/errors"
+	"github.com/go-oauth2/oauth2/v4/manage"
+	"github.com/go-oauth2/oauth2/v4/models"
+	"github.com/go-oauth2/oauth2/v4/server"
+	"github.com/go-oauth2/oauth2/v4/store"
+	pg "github.com/vgarvardt/go-oauth2-pg/v4"
+)
+
+var oauthServer *server.Server
+var clientStore *pg.ClientStore
+
+func SetUpOAuthServer() error {
+	manager := manage.NewDefaultManager()
+	// token store
```

Comment:
Let's keep comments when they are required to understand the code. 

When I read `manager.MustTokenStorage(store.NewMemoryTokenStore())`, I understand "Initialize a new memory token token store and register it to the Oauth manager". The code tells more that the comment. So we can remove the comment :)